openrouteservice-api-tests/src/test/java/org/heigit/ors/v2/services/routing/ResultTest.java

```diff
@@ -1222,6 +1222,29 @@ public void testOptimizedAndTurnRestrictions() {
                 .statusCode(200);
     }
 
+    @Test
+    public void testMaximumSpeedWeighting() {
+        JSONObject body = new JSONObject();
+        body.put("coordinates", constructCoords("8.63348,49.41766|8.6441,49.4672"));
+        body.put("preference", getParameter("preference"));
+        body.put("instructions", true);
+        body.put("user_speed",80);
+
+        // Test that the "right turn only" restriction at the junction is taken into account
```

Comment:
Auxiliary function = helper function. It is cleaner to encapsulate a code section as a new function with a meaningful name than to use a comment, because the code then explains itself.